bin/reth/src/node/mod.rs

```diff
@@ -193,8 +193,8 @@ impl Command {
 
         tx.await??;
 
-        info!(target: "reth::cli", "Finishing up");
-        Ok(())
+        info!(target: "reth::cli", "Pipeline has finished.");
```

Comment:
can we please add a note here why we're doing this in debug.tip mode

Reply:
yeah, sure